src/utils/getSentencesFromText.js

```diff
@@ -0,0 +1,19 @@
+export function getSentencesFromText(text, amount = 0) {
+
+  if (text === '' || null || undefined) return "";
+  text.match(/[!.?]$/) ? text = String(text) : text += '.';
+  text = text.replace(/[\r\n]+/, '')
+  
+  const sentences = text.match( /[^.!?]+[.!?]+/g );
```

Comment:
sentences could be null if not matched. need null-check

Reply:
Done